starlink/Atl.py

```diff
@@ -196,6 +196,10 @@ def readfitswcs( hdu ):
 	    header. For possible values, see the documentation for the
 	    "Encoding" attribute in SUN/211.
 
+         Iwc: Include the Intermediate World Coordinate system in the
+        returned FrameSet, to allow the addition of e.g. a distored
```

Comment:
spelling fixed.

indenting is inconsistent because the file mixes tabs and spaces, with 3-spaces as the typical indent, so it's not clear where to indent to.